java-client/src/test/java/org/opensearch/client/opensearch/experiments/ParsingTests.java

```diff
@@ -88,11 +80,27 @@ public void testFoo() throws Exception {
   }
 
   @Test
-  public void testIndexSettingsMappingParsing() {
+  public void testIndexSettingsSyncIntervalTimeParsing() {
```

Comment:
Could you please add test case to check the deprecated fields (`translog.flush_threshold_size`) are also serialized? Thank you.

Reply:
If we decide to keep serializing the `translog.xxx` fields, I will add the test cases.